README.md

```diff
@@ -175,6 +175,17 @@ REM stop the service
 sc.exe stop DeadlineWorker
 ```
 
+### Running Outside of an Operating System Service
+
+The worker agent can also be started outside of a service context if required. Run `deadline-worker-agent --help` to see a list of supported command line arguments.
+
+**NOTE:** You must have an AWS region specified in order to run the worker agent from the command line. This can be configured through:
+  - Environment Variables:
+    - `AWS_REGION` and `AWS_DEFAULT_REGION`
+  - The AWS CLI:
+    - Run `aws configure` and specify the region.
+  - The [AWS CLI Configuration File](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html). 
```

Comment:
These are kind of related. The CLI just writes to the config file. Perhaps we can make the AWS CLI a sub-bullet of the config file. What do you think?

Reply:
Amended these bullets, let me know what you think